.github/workflows/python-package.yml

```
@@ -0,0 +1,73 @@
+name: Test and deploy on tag
+
+on: push
+
+jobs:
+  test:
+
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        python-version: [3.5, 3.6, 3.7, 3.8, 3.9]
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python ${{ matrix.python-version }}
+      uses: actions/setup-python@v2
+      with:
+        python-version: ${{ matrix.python-version }}
+    - name: Install dependencies
+      run: |
+        python -m pip install --upgrade pip
+        python -m pip install tox tox-gh-actions
+        sudo apt-get update -y
+        sudo apt-get install -y latexmk texlive-latex-extra
+    - name: Run test and build documentation
+      run: tox -e py
```

```suggestion
      run: tox -e test,sphinx2.1,sphinxlatest,codecov
```
This will need some tox changes.

---

eh nevermind, you have them below - not sure they are working though.

---

The test suite was not executed. Fixed in db70f3139eb06c6529163524757cee7e58cfb61b.